gradle/versions.gradle

```diff
@@ -96,5 +96,17 @@ dependencyManagement {
     dependency 'com.github.arteam:simple-json-rpc-client:1.2'
 
     dependency 'org.miracl.milagro.amcl:milagro-crypto-java:0.4.0'
+
+    dependency 'com.zaxxer:HikariCP:3.4.5'
+    dependency 'org.postgresql:postgresql:42.2.16.jre7'
+
+    dependencySet(group: 'org.jdbi', version: '3.14.4') {
+      entry 'jdbi3-core'
+      entry 'jdbi3-sqlobject'
+      entry 'jdbi3-testing'
+    }
+
+    dependency 'com.opentable.components:otj-pg-embedded:0.11.3'
+    dependency 'org.flywaydb:flyway-core:6.1.1'
```

Comment:
what are you using flyway for?

Reply:
for setting up the database tables in the dao unit tests